hw/ip/i2c/doc/_index.md

```diff
@@ -330,6 +343,10 @@ However, all transitions which are dependent on formatting flags are shown expli
 
 ![](I2C_state_diagram.svg)
 
+Similarly, the figure below shows a simplified version of the I2C_Target state machine.
+
+![](I2C_state_diagram_target.svg)
```

Comment:
btw, just a note here, i noticed the diagram uses R/W and ~R/W to distinguish between writes and read... this is a bit confusing when I first looked at it.. since ~R/W reads more like "not read or write" to me.

Reply:
Perhaps, a note below the diagram explaining that "R/W" stands for a R/W bit value would clarify this? The host is reading when R/W bit is "1" and writing when R/W bit is "0".